### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by dgilleland**
Added another folder to ignore in `.gitignore`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # General folder for non-trackable info
 [Nn]o[Tt]rack/
+[Rr]n[Dd]/
```
Comment (dgilleland) [inline .gitignore]: This is my contribution to your course.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # General folder for non-trackable info
 [Nn]o[Tt]rack/
```
Comment (robbinl) [inline .gitignore]: I don't like these lines

Comment (dgilleland) [reply .gitignore]: Well, who do *you* think you are?!

File: .gitignore (hunk 1 above)
Comment (dwelchnait) [inline .gitignore]: Your contribution to this project is acceptable

Comment (dgilleland) [reply .gitignore]: Ok, then why don't you approve it and merge it in?

Comment (dwelchnait) [reply .gitignore]: not my job ...mannn

Comment (dgilleland) [reply .gitignore]: Well, I made it your job, and you *need* me, so buck up!

**Pull request merged**